invisible_cities/types/symbols.py

```diff
@@ -0,0 +1,100 @@
+from enum import auto
+from enum import Enum
+
+from . ic_types import AutoNameEnumBase
+
+
+class BlsMode(AutoNameEnumBase):
+    mean      = auto()
+    median    = auto()
+    scipymode = auto()
+    mode      = auto()
+
+
+class Contiguity(Enum):
+    FACE   = 1.2
```

Comment:
Not a code comment but I don't understand why the all-caps variables get orange coloring.

Reply:
Probably interpreted as global variables by the editor, but I'm not sure.